lib/bazel/options.go

```diff
@@ -22,6 +22,8 @@ type subcommand interface {
 type baseOptions struct {
 	// Bazel's cache directory for this workspace.
 	OutputBase string
+	// Additional flags appended to bazel after other common flags.
+	ExtraFlags []string
```

Comment:
nit: I think these are called "Startup" flags (https://bazel.build/reference/command-line-reference#startup-options) - maybe we should name this var as such (`ExtraStartupFlags`), to disambiguate from "Subcommand" flags (or whatever we call flags that come after the particular bazel command)

Reply:
done